rest/cache.py

```diff
@@ -69,7 +70,7 @@ def get_int_setting(param, default):
     enabled = False
 
 
-def cache():
+def cache(userSpecific=False):
```

Comment:
This is not a good solution as the cache is not shared between users. It implies that n different users accessing the jobs will generate n RPC to slurmctld and this is not what we want. N users accessing the jobs in a short period of time (< jobs cache expiration) should generate only 1 RPC to slurmctld.